spec.html

```diff
@@ -144,6 +146,8 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
           1. <ins>Set _dateTimeFormat_.[[TimeStyle]] to _timeStyle_.</ins>
           1. <ins>Set _dateTimeFormat_.[[HourCycle]] to _hc_.</ins>
         1. <ins>If _dateStyle_ or _timeStyle_ are not *undefined*, then</ins>
+          1. <ins>If any of the Properties from <emu-xref href="#table-datetimeformat-components"></emu-xref> except of `hourCycle`, is defined in _options_,</ins>
```

Comment:
“except of” doesn’t read right to me.
```suggestion
          1. <ins>If any of the Properties from <emu-xref href="#table-datetimeformat-components"></emu-xref>, not including `hourCycle`, is defined in _options_,</ins>
```